### PR TITLE
Font tag

### PR DESCRIPTION
**Pull request by ethan-master-coding**
### Description

Made an entry for the `<font>` tag.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by Dusch4593** [comment]
@ethan-master-coding `font-tag-example.png` needs to go in the `media/` folder.

**Comment by ethan-master-coding** [comment]
@Dusch4593 Fixed that

File: content/html/concepts/tags/terms/font/font.md
Comment (Dusch4593) [inline content/html/concepts/tags/terms/font/font.md:24]:
```suggestion
- `color`: What [color](https://www.codecademy.com/resources/docs/css/colors) the font/text should be. The color can be set to a named [color], hexadecimal or rgb.
- `face`: What font the text should be. For example to have the text be `verdana` the `face` attribute would be like this `face="verdana"`
- `size`: Sets the size of the font/text. It can use absolute units and relative units ranging from 1-7. Absolute units are used by `size="5"` or any other number 1-7. Relative units are relative the size of the `<basefont>` tag (also deprecated). The size attribute would be set to `+5` or `-2`.
```

File: content/html/concepts/tags/terms/font/font.md
Comment (Dusch4593) [inline content/html/concepts/tags/terms/font/font.md:3]:
```suggestion
Description: 'Used to be used to set the font of text. This is now deprecated.'
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@ethan-master-coding left some comments/suggestions for you to check out

**Comment by ethan-master-coding** [comment]
@Dusch4593 All edits look good. I accepted the changes.

File: content/html/concepts/tags/terms/font/font.md
Comment (Dusch4593) [inline content/html/concepts/tags/terms/font/font.md:16]:
```suggestion
The **`<font>`** tag was used to set the font of text. It is not used any more in favor of the [CSS](https://www.codecademy.com/resources/docs/css) [`font`](https://www.codecademy.com/resources/docs/css/typography/font) property instead because the `<font>` tag is now deprecated.
```

**Review by Dusch4593 — APPROVED** [review]
@ethan-master-coding just one small suggestion but LGTM, otherwise 👍🏻 Onward to the next review 😄

**Comment by ethan-master-coding** [comment]
@Dusch4593 All edits look good. I accepted the changes.

**Comment by ethan-master-coding** [comment]
@Dusch4593 To lazy to start a discussion but can I make a `General: Codecademy` entry?

**Comment by Dusch4593** [comment]
> @Dusch4593 To lazy to start a discussion but can I make a `General: Codecademy` entry?

I'm not sure that's something our readers would go to _Docs_ to learn about. Besides, CC has an ["About" page](https://www.codecademy.com/about).

File: content/html/concepts/tags/terms/font/font.md
Comment (KTom101) [inline content/html/concepts/tags/terms/font/font.md:16]:
```suggestion
The **`<font>`** tag was used to set the font characteristics of a text. It is now deprecated. [CSS](https://www.codecademy.com/resources/docs/css) properties like the [`font`](https://www.codecademy.com/resources/docs/css/typography/font) property should be used instead. 
```

File: content/html/concepts/tags/terms/font/font.md
Comment (KTom101) [inline content/html/concepts/tags/terms/font/font.md:3]:
```suggestion
Description: 'Used to be used to set the font characteristics of a text. This is now deprecated.'
```

File: content/html/concepts/tags/terms/font/font.md
Comment (KTom101) [inline content/html/concepts/tags/terms/font/font.md:20]:
```suggestion
Any combination of the following three attributes could be used in the `<font>` tag, at least one attribute is required:
```

File: content/html/concepts/tags/terms/font/font.md
Comment (KTom101) [inline content/html/concepts/tags/terms/font/font.md:23]:
```suggestion
- `face`: What font the text should be. For example, to have the text be `verdana` the `face` attribute would be set to `face="verdana"`
```

**Review by KTom101 — CHANGES_REQUESTED** [review]
@ethan-master-coding Just a few changes from me and I think this will be good to go.

**Comment by ethan-master-coding** [comment]
@KTom101 All edits look good. I accepted the changes.

File: content/html/concepts/tags/terms/font/font.md
Comment (ethan-master-coding) [inline content/html/concepts/tags/terms/font/font.md:20]:
```suggestion
Any combination of the following three attributes could be used in the `<font>` tag, at least any one attribute is required:
```
I feel like it may sound like to some people that 1 is required but it doesn't say it says. Should we change it to this?

File: content/html/concepts/tags/terms/font/font.md
Comment (KTom101) [inline content/html/concepts/tags/terms/font/font.md:23]:
```suggestion
- `face`: What font the text should be. For example, to have the text be `verdana`, the `face` attribute would be set to `face="verdana"`.
```

File: content/html/concepts/tags/terms/font/font.md
Comment (KTom101) [reply content/html/concepts/tags/terms/font/font.md:20]: sure, we could do that.

**Review by KTom101 — APPROVED** [review]
@ethan-master-coding Added one minor change.

**Comment by ethan-master-coding** [comment]
@KTom101 Made the 2 changes

**Comment by ethan-master-coding** [comment]
@KTom101 Can we revert those changes since all the checks are failing now?

**Comment by ethan-master-coding** [comment]
@KTom101 Are we good to merge now that all checks are passing?

**Comment by KTom101** [comment]
@ethan-master-coding Let's see if @Dusch4593 has any more changes before merging.

File: content/html/concepts/tags/terms/font/font.md
Comment (Dusch4593) [inline content/html/concepts/tags/terms/font/font.md:20]:
```suggestion
At least one of the following three attributes is required for the `<font>`. However any combination of them can be used:
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@ethan-master-coding just one last small suggestion and then we should be good to merge after that 👍🏻

**Comment by ethan-master-coding** [comment]
@Dusch4593 All edits look good. I accepted the changes.

**Comment by Dusch4593** [comment]
@ethan-master-coding Looks good. Merging. I also my a tiny tiny change by adding a comma in 69e8aa86630390255630d1c52698a465d7121835.

**Review by Dusch4593 — APPROVED** [review]

**Comment by Dusch4593** [comment]
@ethan-master-coding Thanks for this new entry! When it goes live, we can check it here: https://www.codecademy.com/resources/docs/html/tags/font

**Pull request merged**
